src/components/Visualizations/TotalRequests.jsx

```diff
@@ -65,13 +66,33 @@ const TotalRequests = ({
     },
     tooltips: {
       mode: 'index',
-      callbacks: {
-        footer: (tooltipItem, data) => {
-          const { index } = tooltipItem[0];
-          const total = data.datasets.reduce((p, c) => p + c.data[index], 0);
-          return `Total ${total}`;
-        },
-      },
+      custom: adapter(ttData => {
+        const lines = [];
+        let total = 0;
+
+        ttData.body.forEach((line, idx) => {
+          const [abbrev, requests] = line.lines[0].split(': ');
```

Comment:
This looks wild... can you drop some comments in here to 'splain whats going on?
Or even extract this to its own function if its something that will be modified often

Reply:
Yeah, I know it's a bit ugly...basically for every tooltip we need to convert ChartJS's tooltip data model into some that our own custom tooltip can digest. And their model is pretty awkward. But yeah I can put some comments in there.